homework/CSharpException/Triangle.cs

```diff
@@ -0,0 +1,55 @@
+﻿
+namespace CSharpException
+{
+    public class Triangle
+    {
+        public int OneLength { get; set; }
+        public int TwoLenght { get; set; }
+        public int ThreeLenght { get; set; }
+
+        public Triangle()
+        {
+        }
+        public Triangle(int oneLength, int twoLenght, int threeLenght)
+        {
+            OneLength = oneLength;
+            TwoLenght = twoLenght;
+            ThreeLenght = threeLenght;
+        }
+
+        public void SideLengthCheck(int OneLength, int TwoLenght, int ThreeLenght)
+        {
+            if (OneLength < TwoLenght + ThreeLenght
+                || TwoLenght < OneLength + ThreeLenght
+                || ThreeLenght < OneLength + TwoLenght)
+            {
+                throw new Exception("The length of a side is greater than the sum of the other two!");
+            }
+            else
+            {
+                Console.WriteLine("The length of a side is less than the sum of the other two!");
+            }
+        }
+
+        public void CheckingValuesForZero(int OneLength, int TwoLenght, int ThreeLenght)
+        {
+            do
+            {
+                Console.WriteLine("Enter the lengths of all sides:");
+                OneLength = int.Parse(Console.ReadLine());
```

Comment:
I thing there is something wrong.
First of all if you pass parameters to a method, then they must be initialized outside the class. And you pass them values directly in the method, which is not a good practice. (OneLength = int.Parse(Console.ReadLine());)
The second only check if the number is 0 or not in the method (single responsibility principle).
The third it is better to move loop to the Program class and thing of better condition for while.